e2e/e2e_test.go

```diff
@@ -32,6 +37,14 @@ func init() {
 	fmt.Println("timeout for deploytimeout ", deployTimeout)
 }
 
+func setDefaultKubeconfig() {
+	_, exists := os.LookupEnv("KUBECONFIG")
+	if !exists {
+		defaultKubeconfig := filepath.Join(os.Getenv("HOME"), ".kube", "config")
```

Comment:
This will only work if you are running E2E on the same machine where kubernetes is deployed

it will fail if you want to run E2E against the remote cluster, I think setting `KUBECONFIG` is the best option

Reply:
Hi, @Madhu-1 :) I'm sorry for lack of explanation. I think this code is same what you're saying. It is not related with real kubernetes cluster but it related with kubeconfig file. In our e2e README.md, default kubeconfig is $HOME/.kube/config. This code just set KUBECONFIG envorinment variable as $HOME/.kube/config. Of course, this code works with remote kubernetes cluster.

```
// in our e2e docs...
kubeconfig | Path to kubeconfig containing embedded authinfo (default: $HOME/.kube/config)
```